api/src/main/java/org/cardanofoundation/rosetta/api/construction/service/CardanoConstructionServiceImpl.java

```diff
@@ -276,10 +274,9 @@ public TransactionWitnessSet getWitnessesForTransaction(List<Signatures> signatu
     ArrayList<BootstrapWitness> bootstrapWitnesses = new ArrayList<>();
     log.info("[getWitnessesForTransaction] Extracting witnesses from signatures");
     signaturesList.forEach(signature -> {
-      if (ObjectUtils.isEmpty(signature)) {
+      if (signature == null) {
```

Comment:
Here `signature` could not be null due to the method calls. However, I left the validation here just in case.